deploy/kustomize/base/base.yaml

```
@@ -39,6 +39,20 @@ objects:
     # The bulk of your App. This is where your running apps will live
     deployments:
     -
+    jobs:
+    - name: koku-script-runner-job
+      podSpec:
+        name: koku-script-runner
```

nitpick:
```suggestion
    - name: script-runner
      podSpec:
```
`podSpec` does not take a name.

The combo between the clowdapp and the cji is creating a pod called: `koku-koku-script-runner-job-koku-script-runner-job-*`.

With all of the above suggestions, this name should become: `koku-script-runner-koku-cji-*` which is a little clearer.